### PR TITLE
Update README.afk.md

### PR DESCRIPTION
**Pull request by joshhazelhurst123**
Add the warning "This file has been translated automatically" to all translation files #115

### CODE REVIEW

File: translations/README.afk.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+NOTE: This file has been translated automatically. If you find an error, just make a PR with the edits.
```
Comment (gomzyakov) [inline translations/README.afk.md]:
```suggestion
>**NOTE:* _This file has been translated automatically. If you find an error, just make a PR with the edits._
```

**Comment by gomzyakov** [comment]
@joshhazelhurst123 

I added some formatting to your edits. Otherwise everything is great! You can remove the draft from PR.

**Comment by joshhazelhurst123** [comment]
Add the warning "This file has been translated automatically" to all translation files #115 #140
Let me know how this pull request is. If I need to make any changes to the edits. Cheers

**Comment by gomzyakov** [comment]
@joshhazelhurst123 Hello!

Is this duplicate of https://github.com/good-first-issues/good-first-issues.github.io/pull/140 ?

I suppose so. Close this PR please. I accepted https://github.com/good-first-issues/good-first-issues.github.io/pull/140 ;)

**Comment by joshhazelhurst123** [comment]
its a duplicate. Awesome thank you @gomzyakov

**Pull request closed without merging**